core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -138,6 +139,37 @@ public void testNullStatusOnKraftCommandAlterClientMetrics() {
         assertEquals("Completed updating config for client-metric cm.", message);
     }
 
+    @ClusterTest
+    public void testAddConfigKeyValuesUsingCommand() throws Exception {
+
+        try (Admin client = cluster.admin()) {
+            NewTopic newTopic = new NewTopic("test-topic", 1, (short) 1);
+            client.createTopics(Collections.singleton(newTopic)).all().get();
+            cluster.waitForTopic("test-topic", 1);
+        }
+
+        Stream<String> command = Stream.concat(quorumArgs(), Stream.of(
+                "--entity-type", "topics",
+                "--entity-name", "test-topic",
+                "--alter", "--add-config", "cleanup.policy=[delete,compact]"));
+
+        String message = captureStandardStream(false, run(command));
+        assertEquals("Completed updating config for topic test-topic.", message);
+
+        command = Stream.concat(quorumArgs(), Stream.of(
+                "--entity-type", "topics",
+                "--entity-name", "test-topic",
+                "--describe"));
+
+        message = captureStandardStream(false, run(command));
+        assertTrue(message.contains("cleanup.policy=delete,compact"), "Config entry was not added correctly");
+
+        try (Admin client = cluster.admin()) {
+            client.deleteTopics(Collections.singleton("test-topic")).all().get();
```

Comment:
ditto.
Additionally, we can reuse the same `admin` to avoid create admin two times.

Reply:
Seems good.I will adjust my code.Thanks!

> ditto. Additionally, we can reuse the same `admin` to avoid creating admin two times.